build.sbt

```diff
@@ -25,9 +26,16 @@ lazy val utest = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     }),
 
     unmanagedSourceDirectories in Compile += {
-      val v = if (scalaVersion.value startsWith "2.10.") "scala-2.10" else "scala-2.11"
+      val v = "scala-" + scalaVersion.value.split("\\.").take(2).mkString(".")
```

Comment:
This was needed to make `src/main/scala-2.13` work with 2.13.0-RC1, which we use for `MacroCompat.scala`